### PR TITLE
feat: add commitizen version input

### PR DESCRIPTION
**Pull request by Woile**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -4,6 +4,9 @@ if [ $INPUT_DRY_RUN ]; then INPUT_DRY_RUN='--dry-run'; else INPUT_DRY_RUN=''; fi
 if [ $INPUT_CHANGELOG ]; then INPUT_CHANGELOG='--changelog'; else INPUT_CHANGELOG=''; fi
 if [ $INPUT_PRERELEASE ]; then INPUT_PRERELEASE="--prerelease $INPUT_PRERELEASE"; else INPUT_PRERELEASE=''; fi
 if [ "$INPUT_COMMIT" == 'false' ]; then INPUT_COMMIT='--files-only'; else INPUT_COMMIT=''; fi
+if [ "$INPUT_COMMITIZEN_VERSION" == 'latest' ]; then COMMITIZEN_VERSION=''; else COMMITIZEN_VERSION="commitizen==$INPUT_COMMITIZEN_VERSION"; fi
```
Comment (floredelasteyrie) [inline entrypoint.sh:7]: Shouldn't the case `latest` define `COMMITIZEN_VERSION="commitizen"` ?
Else when we install requirements we have `pip install ''` - I think that's why tests are failing

```suggestion
if [ "$INPUT_COMMITIZEN_VERSION" == 'latest' ]; then COMMITIZEN_VERSION="commitizen"; else COMMITIZEN_VERSION="commitizen==$INPUT_COMMITIZEN_VERSION"; fi
```

**Comment by Woile** [comment]
Not sure why it's failing now 🤔

File: entrypoint.sh [hunk 2]
```diff
@@ -13,46 +16,44 @@ REPOSITORY=${INPUT_REPOSITORY:-$GITHUB_REPOSITORY}
...
+  echo 'Missing input "github_token: ${{ secrets.GITHUB_TOKEN }}".'
+  exit 1
+}
 
 echo "Repository: $REPOSITORY"
 echo "Actor: $GITHUB_ACTOR"
 
 echo "Installing requirements..."
-pip install commitizen $INPUT_EXTRA_REQUIREMENTS
+pip install "$INPUT_COMMITIZEN_VERSION" "$INPUT_EXTRA_REQUIREMENTS"
```
Comment (floredelasteyrie) [inline entrypoint.sh:27]: I tested it locally, seems an issue comes from double quotes here (as default for this variable is `''`) !

```suggestion
pip install "$INPUT_COMMITIZEN_VERSION" $INPUT_EXTRA_REQUIREMENTS
```

File: action.yml [hunk 3]
```diff
@@ -54,3 +54,7 @@ inputs:
     description: 'Email address used to configure git (for git operations)'
     required: false
     default: 'github-actions[bot]@users.noreply.github.com'
+  commitizen_version:
+    description: 'Specify the version to be used by commitizen'
+    required: false
+    default: latest
```
Comment (floredelasteyrie) [inline action.yml:60]: And I'm not sure it's an issue, but to be consistent with other defaults maybe this should be quoted

```suggestion
    default: 'latest'
```

**Pull request merged**